test/new_relic/agent/transaction/segment_test.rb

```diff
@@ -296,7 +296,7 @@ def test_request_attributes_in_agent_attributes
           segment = txn.segments[0]
           actual = segment.attributes.agent_attributes_for(AttributeFilter::DST_SPAN_EVENTS)
 
-          assert_equal "/referered", actual[:'request.headers.referer']
+          assert_equal "/referred", actual[:'request.headers.referer']
```

Comment:
I know that the rfc spec item has a typo (bane of my existence), but this isn't that, so I don't think the spelling here is proper (it wouldn't be anyway...)

Reply:
Agreed! I think the earlier spelling was trying to play off the rfc spec misspelling.